storybook/stories/playground/index.js

```diff
@@ -70,9 +66,16 @@ function App() {
 					</DropZoneProvider>
 				</SlotFillProvider>
 			</div>
-		</Fragment>
+		</>
 	);
 }
 
-registerCoreBlocks();
-render( <App />, document.querySelector( '#app' ) );
+export default {
+	title: 'Playground|Block Editor',
+};
+
+export const _default = () => {
+	registerCoreBlocks();
```

Comment:
I guess it's probably better to wrap in `useEffect( , [])` to only run on mount?

Reply:
The fun part is that this exported function can't take hooks as it isn't a component but some special method that wraps stories. Anyway, we probably move it to `App` and define as an effect there, I will give it a spin 👍